pymatgen/io/lobster/inputs.py

```diff
@@ -156,6 +157,7 @@ def __setitem__(self, key, val):
     def __getitem__(self, item):
         """Implements getitem from dict to avoid problems with cases."""
         found = False
```

Comment:
```suggestion
        new_key = item
        for key_here in self:
            if item.strip().lower() == key_here.lower():
                new_key = key_here
```

Can you replace the while function with this? My code way overly complicated and the None is unnecessar.